crates/oxc_linter/src/snapshots/jsx_no_target_blank.snap

```diff
@@ -0,0 +1,405 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: jsx_no_target_blank
+---
+  ⚠ eslint-plugin-react(jsx-no-target-blank): Using target=`_blank` without rel=`noreferrer` (which implies rel=`noopener`) is a security risk in older browsers: see https://mathiasbynens.github.io/
+  │ rel-noopener/#recommendations
+   ╭─[jsx_no_target_blank.tsx:1:1]
+ 1 │ <a target="_blank" href="https://example.com/1"></a>
```

Comment:
Shall we report on `target="_blank"` ? Seems it's where the actual error occur.

Reply:
![image](https://github.com/oxc-project/oxc/assets/42141272/8883f5b6-b913-4838-90a0-d2916c40a538)
I tested the rule in VSCode, and it seems to also report on open JSX elements.

Reply:
We don't need to keep same with it, do you think it will be easier to understand the error when focus on it?